docs/libcurl/symbols-in-versions

```diff
@@ -709,6 +709,8 @@ CURL_FORMADD_NULL               7.9.8
 CURL_FORMADD_OK                 7.9.8
 CURL_FORMADD_OPTION_TWICE       7.9.8
 CURL_FORMADD_UNKNOWN_OPTION     7.9.8
+CURL_GET_SSLVERSION             7.53.0
+CURL_GET_SSLVERSION_MAX         7.53.0
```

Comment:
When i remove it I have issue with test 1119:
```

test 1119...[Verify that symbols-in-versions and headers are in sync]

perl  returned 2, when expecting 0

 exit FAILED

== Contents of files in the log/ dir after test 1119

=== Start of file stdout1119

 CURL_GET_SSLVERSION

 CURL_GET_SSLVERSION_MAX

 CURL_SET_SSLVERSION_MAX

=== End of file stdout1119
```
And don't find any suffix word for ignore them in a script tests/symbol-scan.pl. Can I add suffix word "_PRIVATE" or "_HELPER" to the script ?